app/models/moderation_report.rb

```diff
@@ -54,4 +54,8 @@ def self.search(params)
 
     q.apply_default_order(params)
   end
+
+  def self.prune!
+    where("created_at < ?", 1.week.ago).delete_all
+  end
```

Comment:
Reports don't need to be pruned. I prefer to keep data unless there's a strong need to delete it.